extension/app/ts/components/pages/ChangeChain.tsx

```diff
@@ -36,25 +36,25 @@ export function ChangeChain() {
 			await updatePage(message)
 		}
 		browser.runtime.onMessage.addListener(popupMessageListener)
-		sendPopupMessageToBackgroundPage( { method: 'popup_changeChainReadyAndListening' } )
+		sendPopupMessageToBackgroundPage({ method: 'popup_changeChainReadyAndListening' })
 		return () => browser.runtime.onMessage.removeListener(popupMessageListener)
-	}, [])
```

Comment:
Will this result in this effect never running?  With `useSignalEffect` it will automatically detect any read signals and re-run the effect anytime any of those signals change.  Does it work the same with `useState` variables, where if they are set it will run again?

Reply:
,[] means run once, no param means run always, so it will always have the newest state available when updated. If you just run it once, the state it has available is the default initial state, which is not general what we want.

I really should look into signals 😬